### DISCUSSION TITLE
how does the Configuration option work?

### DISCUSSION THREAD

**Discussion by essie98927** [UnamSanctam/UnamWebPanel#178] [Q&A]
Hi,

Could someone help me understand a bit more about how the Configuration option works?

Do you put the same data as what you put in the remote config option on the miner?

If yes does it automatically understand if it's XMR or ETC/ETH to not erase another algo options? My miner has XMR and ETC as default algos, and I would like to be able to change through the the panel the ETC algo to ETH without doing anything to the XMR how could I do that?

Thanks a lot if you're taking time to answer me.

**Comment by UnamSanctam** [comment]
Yes they're the same format. You can change the ETH miners algorithm between ethash, etchash and ubqhash without issue, and you can change the XMR miners algorithm to any other algorithm except the ETH ones (since all other algorithms are in the XMR miner). You cannot however change an ETC miner to an XMR miner and vice versa. The ETH miner and XMR miner has different formats so they shouldn't overwrite each other but normally you just make one configuration for ETH and one for XMR, or if you want to change the configuration for all XMR miners then change the "Default xmrig" and if you want to change it for all ETH miners then change the "Default ethminer" configuration.

(XMR miner means xmrig and ETH miner means ethminer).

**Reply by essie98927** [reply]
Got you, will try all that stuff out! 

Thank you very much again for your quick answer, hope you're doing well with future updates.
